easybuild/tools/docs.py

```diff
@@ -707,6 +710,66 @@ def list_software_txt(software, detailed=False):
     return '\n'.join(lines)
 
 
+def gather_reverse_dependencies():
+    """
+    Gather reverse dependencies from installed modules.
+
+    :return: dictionary with reverse dependencies
+    """
+
+    dependencies = {}
+    reverse_dependencies = {}
+
+    modtool = modules_tool()
+
+    available_modules = modtool.available()
+
+    for idx, mod_name in enumerate(available_modules):
+
+        mod_deps = dependencies_for(mod_name, modtool, alldeps=dependencies, full_module_names=True)
+
+        for dep_mod_name in mod_deps:
+            if dep_mod_name not in reverse_dependencies:
+                reverse_dependencies[dep_mod_name] = []
+            reverse_dependencies[dep_mod_name].append(mod_name)
+
+        sys.stdout.write('\r%s of %s modules checked' % (idx+1, len(available_modules)))
+        sys.stdout.flush()
+
+    sys.stdout.write('\n')
+
+    return reverse_dependencies
+
+
+def list_reverse_dependencies(mod_names, output_format=FORMAT_TXT):
+    """
+    Show reverse dependencies of installed software
+
+    :param output_format: output format to use
+    :return: multi-line string presenting requested info
+    """
+
+    all_reverse_deps = gather_reverse_dependencies()
+
+    # TODO: format output
+
+    if mod_names == ['all']:
+        output_mod_names = all_reverse_deps.keys()
+    else:
+        output_mod_names = []
+        for mod_name in mod_names:
+            for full_mod_name in all_reverse_deps:
+                if full_mod_name.startswith(mod_name):
+                    output_mod_names.append(full_mod_name)
+
+    if not output_mod_names:
+        return "No reverse dependencies found for %s" % ', '.join(mod_names)
+    else:
+        return '\n'.join(["%s: %s" % (mod_name, reverse_deps)
+                          for mod_name, reverse_deps in all_reverse_deps.items()
+                          if mod_name in output_mod_names])
```

Comment:
This can be simplified using:

```python
return '\n'.join(["%s: %s" % (m, all_reverse_deps[m]) for m in output_mod_names])
```

We should also ensure sorted output?